src/Appwrite/Platform/Tasks/Retry.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Appwrite\Platform\Tasks;
+
+use Utopia\CLI\Console;
+use Utopia\Platform\Action;
+use Utopia\Queue\Client;
+use Utopia\Queue\Connection;
+use Utopia\Validator\Text;
+
+class Retry extends Action
+{
+    public static function getName(): string
+    {
+        return 'retry';
+    }
+
+
+    public function __construct()
+    {
+        $this
+            ->desc('Retry Queue')
+            ->param('name', '', new Text(128), 'Queue name')
+            ->inject('queue')
+            ->callback(fn ($queueName, $queue) => $this->action($queueName, $queue));
+    }
+
+    public function action(string $queueName, Connection $queue): void
```

Comment:
```suggestion
    public function action(string $name, Connection $queue): void
```
Lets also add a doc block explaining the function and the parameters